e2e/js_image_oci/README.md

```diff
@@ -0,0 +1,50 @@
+# An example/e2e for rules_js + rules_oci
+
+The rule returns `tar` artifacts, suitable to include in the `tars` attribute of the `oci_image` rule from rules_oci.
+
+## Fine-grained layering
+
+`js_image_layer` is a macro that yields two tar files `app.tar` and `node_modules.tar`. While `app.tar` contains first-party sources, `node_modules.tar` contains all third-party dependencies.
+
+This speeds up developer change-build-push cycle by allowing build and push of only what has changed.
+
+For instance, when a new third-party dependency is added, then only `node_modules.tar` will change and one will only have to push changes to dependencies.
+On the other hand, if the application code is changed, then only `app.tar` will be updated and pushed.
+
+## Selecting the right NodeJS interpreter
+
+By default `js_binary` gets the nodejs interpreter for the host platform. However, this is not the case when including the js_binary in js_image_layer thanks to transitions. See [#3373](https://github.com/bazelbuild/rules_nodejs/pull/3373) for how this works.
+
+Toolchain selection is controlled by `platform` attribute on `js_image_layer`.
+NodeJS interpreter for a different platform can be obtained by changing `platform`.
+
+Here is what the final image looks like when the platform is `linux/arm64`
```

Comment:
It would be easier to understand a snippet of starlark where `platform = "linux/arm64"` so that you can see what you type in order to get the final image below

Reply:
+1 one to this nit of being explicit with `platform = "linux/arm64"` instead of "platform is `linux/arm64`"